src/main/java/com/homework/springboot/service/impl/UserRepositoryServiceImpl.java

```
@@ -0,0 +1,37 @@
+package com.homework.springboot.service.impl;
+
+import com.homework.springboot.model.User;
+import com.homework.springboot.repository.UserRepository;
+import com.homework.springboot.service.UserRepositoryService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+@RequiredArgsConstructor
+public class UserRepositoryServiceImpl implements UserRepositoryService {
+
+    private  UserRepository repository ;
+
+    @Override
+    public void saveUser(User user) {
+
+    }
+
+    @Override
+    public void updateUser(User user) {
+
+    }
+
+    @Override
+    public void deleteUser(Long id) {
+
```

The same

---

If the method is not implemented better to throw UnsupportedOperationException